bbb-exporter/collector.py

```diff
@@ -129,3 +139,16 @@ def collect(self):
 
         logging.info("Finished collecting metrics from BigBlueButton API")
 
+    @staticmethod
+    def _get_participant_count_by_client(self, meetings):
+        p_by_c = defaultdict(int, {'HTML5': 0, 'DIAL-IN': 0, 'FLASH': 0})
+        for meeting in meetings:
+            if isinstance(meeting['attendees']['attendee'], list):
```

Comment:
Add some error checking if `meeting['attendees']` exists. When testing this I ended up with a `TypeError: 'NoneType' object is not subscriptable` because `meeting["attendees"]` was `None`.

Steps to reproduce the issue:
1. Make sure the server is empty
1. Start exporter
2. Visit the exporters metrics
3. Open bbb and create a new room
4. Refresh the exporters metrics
5. Close the bbb tab (do not end the meeting, just close the tab)
6. Refresh the exporters metrics